db/models/index.js

```diff
@@ -18,28 +18,23 @@ const Photo = require('./photo');
 const BrewType = require('./brewType');
 const Unit = require('./unit');
 
-  // I'm assuming that the following will be added thru associations (see index).
-  // BrewID... unique brew types... each product can have one
-  // SellerID... one seller per product
-  // Photos... multiple to one product entry
-
 OAuth.belongsTo(User);
 User.hasOne(OAuth);
 
-Product.belongsTo(Seller); //sellerID on product
-Seller.hasMany(Product); //symmetrical
+Product.belongsTo(Seller, {as: 'seller'}); //sellerID on product
```

Comment:
I believe if the alias is the same as the table name, it is redundant (without it, sellerId would still be on Product)